pyproject.toml

```
@@ -17,7 +17,7 @@ include = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.9"
+python = "^3.8"
```

If I'm going to be supporting multiple versions, I'd want to support all active versions. Using [End of Life](https://endoflife.date/python) the oldest is 3.7. (3.6 is still technically not EOL but it's about to be so I don't think it's worth including).
```suggestion
python = "^3.7"
```